src/client/components/Meals/MealReviews.js

```diff
@@ -0,0 +1,165 @@
+import React, { useState, useEffect } from 'react';
+import { useParams } from "react-router";
+import ReactStars from "react-rating-stars-component";
+import Moment from 'moment';
+import Pics from '../Helper/Pics';
+import { Link } from 'react-router-dom';
+import OkModal from '../Helper/OkModal'
+
+const Mealreviews = (props) => {
```

Comment:
Try to keep your naming consistent -> MealReviews

Reply:
fixed it